content/en/docs/concepts/storage/storage-capacity.md

```diff
@@ -0,0 +1,117 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Storage Capacity
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+Storage capacity is limited and may vary depending on the node on
+which a pod runs: network-attached storage might not be accessible by
+all nodes, or storage is local to a node to begin with.
+
+{{< feature-state for_k8s_version="v1.19" state="alpha" >}}
+
+This page describes how Kubernetes keeps track of storage capacity and
+how the scheduler uses that information to schedule Pods onto nodes
+that have access to enough storage capacity for the remaining missing
+volumes. Without storage capacity tracking, it is random whether the
+selected node can run the Pod and multiple scheduling retries may be
```

Comment:
be more specific: "whether the selected node has enough storage requested by the Pod"

Reply:
"requested by the Pod" is potentially misleading for a claim reference: then it's the PVC which requests storage, not the Pod.

I went with "whether the selected node has enough storage for the Pod".